fluster/main.py

```diff
@@ -34,6 +34,12 @@
 RESULTS_DIR = "results"
 
 
+def fluster_main():
```

Comment:
```suggestion
def fluster_main() -> None:
```
flake is complaining about this. could you modify and push again? thanks